### ISSUE TITLE
Peroneal tendon subluxation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11167]
**Preferred term label:**
Peroneal tendon subluxation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Dynamic ultrasound of the lateral ankle showing subluxation or dislocation of one or both of the peroneal tendons out of the peroneal groove

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peroneal tendon disorder: https://www.ncbi.nlm.nih.gov/books/NBK544354/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001408
- label: Peroneal tendon subluxation
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:31335074
- def: Dynamic ultrasound of the lateral ankle showing subluxation or dislocation of one or both of the peroneal tendons out of the peroneal groove.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11167
- synonyms:
